presto-main/src/main/java/com/facebook/presto/metadata/StaticFunctionNamespace.java

```diff
@@ -700,7 +700,7 @@ public FunctionMetadata getFunctionMetadata(FunctionHandle functionHandle)
         return new FunctionMetadata(signature.getName(), signature.getArgumentTypes(), signature.getReturnType(), signature.getKind(), function.isDeterministic(), function.isCalledOnNullInput());
     }
 
-    public FunctionHandle lookupFunction(QualifiedName name, List<TypeSignatureProvider> parameterTypes)
+    public StaticFunctionHandle lookupFunction(QualifiedName name, List<TypeSignatureProvider> parameterTypes)
```

Comment:
Same reason as above. `FunctionHandle` looks cleaner?

Reply:
`lookupFunction` won't be in `FunctionNamespace` interface. But in general I agree with your point.